test/framework/vsphere.go

```diff
@@ -114,11 +114,6 @@ func NewVSphere(t *testing.T, opts ...VSphereOpt) *VSphere {
 	}
 
 	v.cidr = os.Getenv(cidrVar)
-	clusterIP, err := PopIPFromEnv(ClusterIPPoolEnvVar)
```

Comment:
@g-gaston , I moved this back to ClusterConfigFillers otherwise only 1 ip address gets set for all clusters in a multi cluster test since the provider is created once and passed to each cluster test.

Reply:
In that case I would vote to create multiple copies of the provider instead
I will start thinking how to make those tests better but I'm not sure the provider can be shared with the current design